resources/views/waitingrooms/edit.blade.php

```diff
@@ -0,0 +1,36 @@
+@extends('layouts.master')
+
+@section('title', 'Edit Waiting Room ' . $room->id)
+
+@section('main_content')
+
+<div class='container__block'>
+    <div class='wrapper'>
+        {!! Form::model($room, ['method' => 'PATCH','route' => ['waitingrooms.update', $room->id]]) !!}
+            <div class="form-item">
+                {{ Form::label('campaign_id', 'Campaign ID (node id):', array('class' => 'field-label')) }}
+                {{ Form::text('campaign_id', NULL, array('class' => 'text-field')) }}
+            </div>
+
+            <div class="form-item">
+                {{ Form::label('campaign_run_id', 'Campaign Run ID:', array('class' => 'field-label')) }}
+                {{ Form::text('campaign_run_id', NULL, array('class' => 'text-field')) }}
+            </div>
+
+            <div class="form-item">
+                {!! Form::label('signup_start_date', 'Signup Start Date: ', array('class' => 'field-label')) !!}
+                {!! Form::date('signup_start_date', $room->signup_start_date, ['class' => 'text-field']) !!}
+            </div>
+
+            <div class="form-item">
+                {!! Form::label('signup_end_date', 'Signup End Date: ', array('class' => 'field-label')) !!}
+                {!! Form::date('signup_end_date', $room->signup_end_date, ['class' => 'text-field']) !!}
```

Comment:
I'm not sure what @weerd and did is best practice or if it's allowed
but since this stuff is the same across create/update, [we included a form partial](https://github.com/DoSomething/longshot/blob/master/app/views/application/partials/_form_application.blade.php) for all of these so you don't have to copy and paste and add new fields in two places

cc @DFurnes do you have thoughts on partials like that?